### PR TITLE
Remove all {% raw %} usage

### PR DESCRIPTION
**Pull request by wfjvdham**
Removes the {% raw %} usage and replace them by `v-text`

closes https://github.com/lnbits/lnbits/issues/2192

### CODE REVIEW

**Comment by prusnak** [comment]
@wfjvdham have you tested all screens that the change works as intended?

**Review by dni — APPROVED** [review]
Thanks for you contribution!

utACK

**Comment by prusnak** [comment]
> No not yet, do you know a good way to be sure te check them all?

No, I don't. Only manual testing, there are not many screens you changed.

**Comment by dni** [comment]
> @wfjvdham have you tested all screens that the change works as intended?
> 
> I remember we had some issues with `q-tooltip` in the past and ended up doing
> 
> ```
> <q-tooltip><span v-text="$t('coming_soon')"></span></q-tooltip>
> ```
> 
> instead of
> 
> ```
> <q-tooltip v-text="$t('coming_soon')"></q-tooltip>
> ```
> 
> Also this part feels wrong, because `v-text` in `q-btn` will nuke the `q-tooltip` completely, no?
> Screenshot 2024-03-12 at 11 48 12

good spot, this is true

**Review by talvasconcelos — APPROVED** [review]
commited some changes! 

looks ok!

File: lnbits/core/templates/core/extensions.html [hunk 1]
```diff
@@ -239,8 +235,7 @@
             v-if="extension.isInstalled && extension.installedRelease"
             class="float-right"
           >
-            <q-badge>
-              {% raw %}{{ extension.installedRelease.version }}{% endraw %}
+            <q-badge v-text="extension.installedRelease.version">
```
Comment (prusnak) [inline lnbits/core/templates/core/extensions.html:238]: I think this one will nuke the `q-tooltip` inside

File: lnbits/core/templates/core/extensions.html [hunk 2]
```diff
@@ -264,44 +259,55 @@ <h6 class="q-my-md text-primary">{%raw%}{{ $t('warning') }}{%endraw%}</h6>
...
       </div>
     </q-card>
   </q-dialog>
 
   <q-dialog v-model="showDropDbDialog">
     <q-card v-if="selectedExtension" class="q-pa-lg">
-      <h6 class="q-my-md text-primary">{%raw%}{{ $t('warning') }}{%endraw%}</h6>
-      <p>{%raw%}{{ $t('extension_db_drop_warning') }}{%endraw%} <br /></p>
+      <h6 class="q-my-md text-primary" v-text="$t('warning')"></h6>
+      <p v-text="$t('extension_db_drop_warning')"><br /></p>
```
Comment (prusnak) [inline lnbits/core/templates/core/extensions.html:288]: This one nukes the `<br>` inside

File: lnbits/core/templates/node/_tab_channels.html [hunk 3]
```diff
@@ -172,11 +171,15 @@
...
-                      <div class="text-caption" v-if="props.row.short_id">
-                        {{ props.row.short_id }}
+                      <div
+                        class="text-subtitle1 col-grow"
+                        v-text="props.row.name"
+                      ></div>
+                      <div
+                        class="text-caption"
+                        v-if="props.row.short_id"
+                        v-text="props.row.short_id"
```
Comment (prusnak) [inline lnbits/core/templates/node/_tab_channels.html:181]: This one nukes the `q-btn` inside

File: lnbits/core/templates/admin/_tab_security.html [hunk 4]
```diff
@@ -130,9 +130,7 @@ <h6 class="q-my-none" v-text="$t('security_tools')"></h6>
               ref="logScroll"
               style="padding: 10px; color: #fafafa; height: 320px"
             >
-              <small v-for="log in logs"
-                >{% raw %}{{ log }}{% endraw %}<br
-              /></small>
+              <small v-for="log in logs" v-text="log"><br /></small>
```
Comment (prusnak) [inline lnbits/core/templates/admin/_tab_security.html:133]: This one nukes the `<br>` inside

File: lnbits/core/templates/core/extensions.html (hunk 1 above)
Comment (wfjvdham) [reply lnbits/core/templates/core/extensions.html]: moved to span

**Review by dni — APPROVED** [review]

**Review by dni — CHANGES_REQUESTED** [review]
formatting @talvasconcelos

**Review by prusnak — APPROVED** [review]
utACK

**Review by dni — APPROVED** [review]

**Pull request merged**